charts/minio/templates/deployment.yaml

```diff
@@ -78,7 +82,11 @@ spec:
         args:
         - /bin/sh
         - -c
+        {{- if .Values.minio.minioCertificates }}
+        - mc --insecure config host add src https://127.0.0.1:9000 "$MINIO_ACCESS_KEY" "$MINIO_SECRET_KEY" && while true; do sleep 1h; done
```

Comment:
You have the certificate mounted already, right? So why do we need `--insecure` here?